src/mem/port.hh

```diff
@@ -438,71 +448,79 @@ class SlavePort : public Port, public AtomicResponseProtocol,
     }
 };
 
+class M5_DEPRECATED SlavePort : public ResponsePort
+{
+    public:
+        SlavePort(const std::string& name, SimObject* _owner,
```

Comment:
Four spaces, not eight :)
```suggestion
    SlavePort(const std::string& name, SimObject* _owner,
```

Reply:
Got it! How do we indent lines where we're past the character limit for a line? For example, like the line below this comment.

Reply:
Very carefully ;). 

I suggest looking within in the file or at other files in the directory and copying that style.

Generally, I would align the next line with the parentheses. The main exception to this would be for `if` statements where you can't tell what is the body and what is the continuation of the condition. Another time this doesn't work is if the parenthesis starts at the end of the line (e.g., >60 chars). 

It's a judgement call... Basically, try to do something that looks the least bad :). The goal is readability. Remember, code is going to be read 10x more than it's written.